Amazon.QLDB.Driver/retry/RetryHandler.cs

```diff
@@ -91,19 +88,6 @@ public T RetriableExecute<T>(Func<T> func, RetryPolicy retryPolicy, Action newSe
                     throw last;
```

Comment:
Since you got rid of the other last, looking at the logic I think you can just throw iex. We dont need "last" nor line 59.

Reply:
Done.